api/candlepin-api-config.json

```diff
@@ -4,5 +4,6 @@
   "invokerPackage": "org.candlepin",
   "groupId": "org.candlepin",
   "artifactId": "candlepin-api",
-  "java8": true
+  "java8": true,
+  "dateLibrary": "java8"
```

Comment:
@kahowell @nikosmoum 
If I remember correctly @kahowell mentioned that dateLibrary value set at openApiGenerate section in build.gradle. 
Do we need this here also?

Reply:
You are right, this is not needed here, so I'll remove it.